client/my-sites/plan-features/footer.jsx

```diff
@@ -12,20 +12,28 @@ import Button from 'components/button';
 import Gridicon from 'components/gridicon';
 
 const PlanFeaturesFooter = ( { translate, current = false, available = true, description, onUpgradeClick = noop } ) => {
+	let upgradeButton;
+
+	if ( current ) {
+		upgradeButton = (
+			<Button className="plan-features__footer-button is-current" disabled>
+				<Gridicon size={ 18 } icon="checkmark" />
+				{ translate( 'Your plan' ) }
+			</Button>
+		);
+	} else if ( available ) {
+		upgradeButton = (
+			<Button className="plan-features__footer-button" onClick={ onUpgradeClick } primary>
+				{ translate( 'Upgrade' ) }
+			</Button>
+		);
+	}
+
 	return (
 		<footer className="plan-features__footer">
 			<p className="plan-features__footer-desc">{ description }</p>
-			<div className="plan-features__footer-buttons">
-				{
-					current
-						? <Button className="plan-features__footer-button is-current" disabled>
-							<Gridicon size={ 18 } icon="checkmark" />
-							{ translate( 'Your plan' ) }
-						</Button>
-						: <Button className="plan-features__footer-button" { ...( available ? { onClick: onUpgradeClick, primary: true } : { disabled: true } ) }>
-							{ translate( 'Upgrade' ) }
-						</Button>
-				}
+			<div className={ 'plan-features__footer-buttons' + ( ! current && ! available ? ' is-empty' : '' ) } >
```

Comment:
It's better to use `classNames` util here because the string concatenation is not very nice and can introduce subtle bugs.

Here's a sample code how it can be accomplished:

```js
import classNames from 'classnames';
....
const buttonsClasses = classNames( 'plan-features__footer-buttons', {
	'is-empty': ! current && ! available
} );
....

<div className={ buttonsClasses }>
```